goruntime_test.go

```diff
@@ -22,8 +24,7 @@ func TestGoTools_Build_File(t *testing.T) {
 	build := Go().Build("main.go").
 		ChangeDir("testingbuild").
 		OutputFilename("blah.exe").
-		ForceRebuild().
-		RaceDetector()
+		ForceRebuild()
```

Comment:
Running this test on a Windows machine without a C compiler will result in a test failure, since the `-race` flag requires cgo. Not a desirable trait.